src/app/app.module.js

```diff
@@ -0,0 +1,31 @@
+(function(angular) {
+  'use strict';
+
+  angular.module('esn-frontend-inbox-calendar', [
+    'esn.session',
+    'esn.notification',
+    'esn.i18n',
+    'op.dynamicDirective',
+    'esn.calendar',
+    'linagora.esn.resource'
+  ]);
+})(angular);
+
+require('esn-frontend-common-libs/src/frontend/js/modules/session.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/notification.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/i18n/i18n.module.js');
+
+require('esn-frontend-calendar/src/linagora.esn.calendar/app/app.js');
```

Comment:
You are requiring all the calendar application files in inbox. You need, in esn-frontend-calendar, to isolate the files/angular modules that you need to import only them in inbox.

Reply:
I split the common part to the libs module